src/interactivity/base/HostSignalInputThread.cpp

```diff
@@ -15,13 +15,13 @@ using namespace Microsoft::Console::Interactivity;
 // - Creates the PTY Signal Input Thread.
 // Arguments:
 // - hPipe - a handle to the file representing the read end of the Host Signal pipe.
-HostSignalInputThread::HostSignalInputThread(_In_ wil::unique_hfile&& hPipe) :
+HostSignalInputThread::HostSignalInputThread(wil::unique_hfile&& hPipe) :
     _hFile{ std::move(hPipe) },
     _hThread{},
-    _dwThreadId{ 0 },
-    _consoleConnected{ false }
+    _dwThreadId{}
 {
     THROW_HR_IF(E_HANDLE, _hFile.get() == INVALID_HANDLE_VALUE);
+    THROW_HR_IF(E_HANDLE, _hFile.get() == nullptr);

```

Comment:
huh, i thought that these were the same

moreover, however: `!_hFile` is the valid way to check for presence in a wil handle